static/css/footer.less

```diff
@@ -189,133 +125,118 @@
         }
 
-        .footer_address {
-            .respond-to-min(1200px, {
-                margin-bottom: unit(30px / @base-font-size-px, em);
-            });
-            .respond-to-range(768px, 1199px, {
-                .grid_column(3);
-            });
-        }
-
-        .footer_general-inqueries {
-            .respond-to-range(768px, 1199px, {
-                .grid_column(3);
-            });
-        }
-    }
+        .respond-to-range(768px, 1199px,{
+            .grid_column(8);
+            border-right: 1px solid @gray-50;
 
-    &-right {
-        .respond-to-max(767px, {
-            display: none;
+            .footer_col {
+                .grid_column(6);
+            }
         });
 
-        .respond-to-range(768px, 1199px, {
+        .respond-to-min(1200px, {
+            .list__horizontal();
             padding-bottom: unit(30px / @base-font-size-px, em);
             margin-bottom: unit(30px / @base-font-size-px, em);
             border-bottom: 1px solid @gray-50;
-        });
 
-        .respond-to-min(1200px, {
-            .grid_column(8);
-            .grid_push(4);
-            border-left-width: @grid_gutter-width;
-            border-right-width: @grid_gutter-width;
+            .footer_col, .footer_list {
+                display: inline;
+            }
+
+            .list_item {
+                margin-right: 2em;
+            }
         });
+    }
 
-        .footer_nested-col-group {
-            .respond-to-min(768px, {
-                .grid_nested-col-group();
-            });
-        }
+    &-middle-right {
+        .respond-to-range(768px, 1199px,{
+            .grid_column(4);
 
-        .footer_col {
-            .grid_column(3);
-        }
+            .footer_list {
+                padding-left: @grid_gutter-width;
+                padding-right: @grid_gutter-width;
+            }
+        });
 
-        .h6 {
-            color: @gray-50;
-        }
+        .respond-to-min(1200px, {
+            .list__horizontal();
+            float: left;
+            width: 50%;
 
-        .list {
-            .respond-to-min(768px, {
-                margin-bottom: 0;
-            });
-        }
+            .list_item {
+                margin-right: 2em;
+            }
+        });
     }
 
     &-post {
-        .respond-to-min(768px, {
+        .respond-to-range(768px, 1199px, {
             padding-top: unit(30px / @base-font-size-px, em);
             margin-top: unit(30px / @base-font-size-px, em);
             border-top: 1px solid @gray-50;
+
         });
+
+        .respond-to-min(1200px, {
+            float: right;
+        });
+
+        .footer_official-website {
+            .short-desc();
+            padding-left: 37px;
+            background: url('../img/us-flag_22x13.png') no-repeat 0 4px;
+
+            .respond-to-max(479px, {
+                max-width: 220px;
+            });
+
+            @media (-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi) {
```

Comment:
I forgot about cf-enhancements. I'll go ahead and add that in this PR since I'm sure it will come up again.